functions/src/index.ts

```diff
@@ -8,10 +9,113 @@ import { initIndex, profileToAlgolia, removeObject } from './lib/algolia'
 import { generateSocialCover } from './lib/migrations'
 import { firestore as db, admin } from './firebase'
 import { notifySlackAboutEvents, notifySlackAboutUsers } from './lib/slack'
+import { getInstagram } from './lib/browser'
+import config from './env'
+import { wrap } from './sentry'
+
+Sentry.init(config.sentry)
 
 const app = express()
 app.use(cors({ origin: true }))
 
+app.post('/import/:source', async (req, res) => {
+  const source = req.params.source
+
+  if (source !== 'instagram') {
+    return res.status(400).json({
+      success: false,
+      error: 'Source not supported',
+      code: 'source_not_supported',
+    })
+  }
+
+  const username = req.body['username'].toLowerCase()
+
+  const profilesByUsername = await db
+    .collection('profiles')
+    .where('username', '==', username)
```

Comment:
`===` or `==` ?

Reply:
That's not a javascript operator. That's a string, second parameter of the `where` function.
Please check [API](https://firebase.google.com/docs/firestore/query-data/get-data#get_multiple_documents_from_a_collection)